spec/wizards/placements/add_placement/steps/additional_subjects_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe Placements::AddPlacement::Steps::AdditionalSubjects, type: :model do
+  describe "attributes" do
+    it { is_expected.to have_attributes(school: nil, parent_subject_id: nil, additional_subject_ids: []) }
+  end
+
+  describe "validations" do
+    it { is_expected.to validate_presence_of(:school) }
+    it { is_expected.to validate_presence_of(:parent_subject_id) }
+    it { is_expected.to validate_presence_of(:additional_subject_ids) }
+
+    describe "#additional_subjects_valid" do
+      let(:school) { create(:placements_school) }
+      let(:parent_subject) { create(:subject) }
+
+      it "is invalid of the subject does not exist" do
+        step = described_class.new(parent_subject_id: parent_subject.id, additional_subject_ids: ["not_a_subject"])
+
+        expect(step).to be_invalid
+      end
+
+      it "is valid if all additional subjects exist" do
+        child_subject = create(:subject, parent_subject:)
+        step = described_class.new(school:, parent_subject_id: parent_subject.id, additional_subject_ids: [child_subject.id])
+
+        expect(step).to be_valid
+      end
+    end
+  end
+
+  describe "#additional_subjects_for_selection" do
+    let(:parent_subject) { create(:subject) }
+    let(:child_subject) { create(:subject, parent_subject:) }
+
+    it "returns additional subjects for the subject" do
```

Comment:
This might be more descriptive:

```suggestion
    it "returns child subjects of the parent subject" do
```